### PR TITLE
Adding recore keyboard

### PR DESCRIPTION
**Pull request by joliverMI**
-Added a new keyboard under PMK: recore

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [ x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by waffle87 — APPROVED** [review]

File: keyboards/pmk/recore/v3/info.json [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+                {"matrix": [4,5], "x": 0.1, "y": 5.34, "w": 1.25},
+                {"matrix": [4,6], "x": 0.65, "y": 4.75, "h": 1.25},
+                {"matrix": [4,7], "x": 0.85, "y": 4.35, "h": 1.25},
+                {"matrix": [9,0], "x": 0, "y": 17.35, "h": 1.25},
+                {"matrix": [9,1], "x": 0, "y": 18.35, "h": 1.25},
+                {"matrix": [9,2], "x": 0, "y": 19.35, "w": 1.25},
+                {"matrix": [9,4], "x": 14.5, "y": 4.25, "w": 1.25},
+                {"matrix": [9,5], "x": 15.75, "y": 4.25, "w": 1.25},
+                {"matrix": [9,6], "x": 17, "y": 4.75},
+                {"matrix": [9,7], "x": 18, "y": 4.75, "w": 1.25},
```
Comment (dunk2k) [inline keyboards/pmk/recore/v3/info.json:130]:
```suggestion
                {"matrix": [0,0], "x": 0, "y": 0.75, "w": 1.25},
                {"matrix": [0,1], "x": 1.25, "y": 0.75},
                {"matrix": [0,2], "x": 2.25, "y": 0.2},
                {"matrix": [0,3], "x": 3.25, "y": 0},
                {"matrix": [0,4], "x": 4.25, "y": 0.2},
                {"matrix": [0,5], "x": 5.25, "y": 0.3},
                {"matrix": [0,6], "x": 6.25, "y": 0.55, "h": 1.25},
                {"matrix": [0,7], "x": 7.25, "y": 0.55, "h": 1.25},

                {"matrix": [5,0], "x": 9.25, "y": 0.55, "h": 1.25},
                {"matrix": [5,1], "x": 10.25, "y": 0.55, "h": 1.25},
                {"matrix": [5,2], "x": 11.25, "y": 0.3},
                {"matrix": [5,3], "x": 12.25, "y": 0.2},
                {"matrix": [5,4], "x": 13.25, "y": 0},
                {"matrix": [5,5], "x": 14.25, "y": 0.2},
                {"matrix": [5,6], "x": 15.25, "y": 0.75},
                {"matrix": [5,7], "x": 16.25, "y": 0.75, "w": 1.25},

                {"matrix": [1,0], "x": 0, "y": 1.75, "w": 1.25},
                {"matrix": [1,1], "x": 1.25, "y": 1.75},
                {"matrix": [1,2], "x": 2.25, "y": 1.2},
                {"matrix": [1,3], "x": 3.25, "y": 1},
                {"matrix": [1,4], "x": 4.25, "y": 1.2},
                {"matrix": [1,5], "x": 5.25, "y": 1.3},
                {"matrix": [1,6], "x": 6.25, "y": 1.8, "h": 1.25},
                {"matrix": [1,7], "x": 7.25, "y": 1.8, "h": 1.25},

                {"matrix": [6,0], "x": 9.25, "y": 1.8, "h": 1.25},
                {"matrix": [6,1], "x": 10.25, "y": 1.8, "h": 1.25},
                {"matrix": [6,2], "x": 11.25, "y": 1.3},
                {"matrix": [6,3], "x": 12.25, "y": 1.2},
                {"matrix": [6,4], "x": 13.25, "y": 1},
                {"matrix": [6,5], "x": 14.25, "y": 1.2},
                {"matrix": [6,6], "x": 15.25, "y": 1.75},
                {"matrix": [6,7], "x": 16.25, "y": 1.75, "w": 1.25},

                {"matrix": [2,0], "x": 0, "y": 2.75, "w": 1.25},
                {"matrix": [2,1], "x": 1.25, "y": 2.75},
                {"matrix": [2,2], "x": 2.25, "y": 2.2},
                {"matrix": [2,3], "x": 3.25, "y": 2},
                {"matrix": [2,4], "x": 4.25, "y": 2.2},
                {"matrix": [2,5], "x": 5.25, "y": 2.3},

                {"matrix": [7,2], "x": 11.25, "y": 2.3},
                {"matrix": [7,3], "x": 12.25, "y": 2.2},
                {"matrix": [7,4], "x": 13.25, "y": 2},
                {"matrix": [7,5], "x": 14.25, "y": 2.2},
                {"matrix": [7,6], "x": 15.25, "y": 2.75},
                {"matrix": [7,7], "x": 16.25, "y": 2.75, "w": 1.25},

                {"matrix": [3,0], "x": 0, "y": 3.75, "w": 1.25},
                {"matrix": [3,1], "x": 1.25, "y": 3.75},
                {"matrix": [3,2], "x": 2.25, "y": 3.2},
                {"matrix": [3,3], "x": 3.25, "y": 3},
                {"matrix": [3,4], "x": 4.25, "y": 3.2},
                {"matrix": [3,5], "x": 5.25, "y": 3.3},
                {"matrix": [3,6], "x": 6.25, "y": 3.05, "h": 1.25},
                {"matrix": [3,7], "x": 7.25, "y": 3.05, "h": 1.25},

                {"matrix": [8,0], "x": 9.25, "y": 3.05, "h": 1.25},
                {"matrix": [8,1], "x": 10.25, "y": 3.05, "h": 1.25},
                {"matrix": [8,2], "x": 11.25, "y": 3.3},
                {"matrix": [8,3], "x": 12.25, "y": 3.2},
                {"matrix": [8,4], "x": 13.25, "y": 3},
                {"matrix": [8,5], "x": 14.25, "y": 3.2},
                {"matrix": [8,6], "x": 15.25, "y": 3.75},
                {"matrix": [8,7], "x": 16.25, "y": 3.75, "w": 1.25},

                {"matrix": [4,0], "x": 0, "y": 4.75, "w": 1.25},
                {"matrix": [4,1], "x": 1.25, "y": 4.75},
                {"matrix": [4,2], "x": 2.25, "y": 4.2, "w": 1.25},
                {"matrix": [4,3], "x": 3.5, "y": 4.2, "w": 1.25},
                {"matrix": [4,5], "x": 4.75, "y": 4.3, "w": 1.25},
                {"matrix": [4,6], "x": 6, "y": 4.8, "h": 1.25},
                {"matrix": [4,7], "x": 7, "y": 5, "h": 1.25},

                {"matrix": [9,0], "x": 9.5, "y": 5, "h": 1.25},
                {"matrix": [9,1], "x": 10.5, "y": 4.8, "h": 1.25},
                {"matrix": [9,2], "x": 11.5, "y": 4.3, "w": 1.25},
                {"matrix": [9,4], "x": 12.75, "y": 4.2, "w": 1.25},
                {"matrix": [9,5], "x": 14, "y": 4.2, "w": 1.25},
                {"matrix": [9,6], "x": 15.25, "y": 4.75},
                {"matrix": [9,7], "x": 16.25, "y": 4.75, "w": 1.25}
```

**Review by dunk2k — CHANGES_REQUESTED** [review]
fixes QMK configurator render

File: keyboards/pmk/recore/v3/config.h [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+#define SERIAL_USART_TX_PIN GP1
+#define SERIAL_USART_RX_PIN GP0
+
+// Communication Settings
+#define FORCED_SYNC_THROTTLE_MS 1000
+#define SPLIT_MAX_CONNECTION_ERRORS 5
+#define SPLIT_CONNECTION_CHECK_TIMEOUT 2000
+
+// RGB Configuration
+#define RGBLIGHT_EFFECT_BREATHE_CENTER 2.0 //1.0 to 2.7
```
Comment (dunk2k) [inline keyboards/pmk/recore/v3/config.h:18]:
```suggestion
#define RGBLIGHT_EFFECT_BREATHE_CENTER 2.0 //1.0 to 2.7

```

File: keyboards/pmk/recore/v3/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [_QWERTY] = LAYOUT(
+//┌────────┬────────┬────────┬────────┬────────┬────────┬── L1 ──┬── X1 ──┐┌── X1 ──┬── L1 ──┬────────┬────────┬────────┬────────┬────────┬────────┐
+   KC_GRV  ,KC_1    ,KC_2    ,KC_3    ,KC_4    ,KC_5   ,MO(_SYMB),MO(_NAV),MO(_NAV),MO(_SYMB),KC_6    ,KC_7    ,KC_8    ,KC_9    ,KC_0    ,KC_MINS ,
+//├────────┼────────┼────────┼────────┼────────┼────────┼── X2 ──┼── X3 ──┤├── X3 ──┼── X2 ──┼────────┼────────┼────────┼────────┼────────┼────────┤
+   KC_TAB  ,KC_Q    ,KC_W    ,KC_E    ,KC_R    ,KC_T    ,KC_LBRC ,KC_MPLY, KC_PAUS ,KC_RBRC , KC_Y    ,KC_U    ,KC_I    ,KC_O    ,KC_P    ,KC_EQL  ,
+//├────────┼────────┼────────┼────────┼────────┼────────┴────────┴────────┤├────────┼────────┼────────┼────────┼────────┼────────┴────────┴────────┤
+   KC_ESC  ,KC_A    ,KC_S    ,KC_D    ,KC_F    ,KC_G    ,                                     KC_H    ,KC_J    ,KC_K    ,KC_L    ,KC_SCLN ,KC_QUOT ,
+//├────────┼────────┼────────┼────────┼────────┼────────┬────────┬────────┤├────────┬────────┬────────┼────────┼────────┼────────┼────────┼────────┤
```
Comment (dunk2k) [inline keyboards/pmk/recore/v3/keymaps/default/keymap.c:21]:
```suggestion
//├────────┼────────┼────────┼────────┼────────┼────────┴────────┴────────┘└────────┼────────┼────────┼────────┼────────┼────────┴────────┴────────┤
   KC_ESC  ,KC_A    ,KC_S    ,KC_D    ,KC_F    ,KC_G    ,                                     KC_H    ,KC_J    ,KC_K    ,KC_L    ,KC_SCLN ,KC_QUOT ,
//├────────┼────────┼────────┼────────┼────────┼────────┬────────┬────────┐┌────────┬────────┬────────┼────────┼────────┼────────┼────────┼────────┤
```
Could this illustrative correction be applied to all other occurrences in both `default` and `via` keymaps please?

File: keyboards/pmk/recore/v3/keymaps/default/keymap.c [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+//├────────┼────────┼────────┼────────┼────────┼────────┴────────┴────────┤├────────┼────────┼────────┼────────┼────────┼────────┴────────┴────────┤
+   KC_TRNS ,KC_LEFT ,KC_LEFT ,KC_DOWN ,KC_RIGHT,KC_RIGHT,                                     KC_LEFT ,KC_LEFT ,KC_DOWN ,KC_RIGHT,KC_RIGHT,KC_NO   ,
+//├────────┼────────┼────────┼────────┼────────┼────────┬────────┬────────┤├────────┬────────┬────────┼────────┼────────┼────────┼────────┼────────┤
+   KC_TRNS ,KC_NO   ,KC_NO   ,KC_DOWN ,KC_NO   ,KC_NO   ,KC_TRNS ,KC_TRNS , KC_TRNS ,KC_TRNS ,KC_NO   ,KC_NO   ,KC_DOWN ,KC_NO   ,KC_NO   ,KC_NO   ,
+//├────────┼────────┼────────┼────────┼────────┼────────┼────────┼────────┘└────────┼────────┼────────┼────────┼────────┼────────┼────────┼────────┤
+   KC_TRNS ,KC_NO   ,KC_NO   ,KC_TRNS ,KC_TRNS ,KC_NO   ,KC_NO   ,                   KC_NO   ,KC_NO   ,KC_NO   ,KC_NO   ,KC_TRNS ,KC_NO   ,QK_CLEAR_EEPROM   
+//└────────┴────────┴── T1 ──┴── T2 ──┴── T3 ──┴── T4 ──┴── T5 ──┘                  └── T5 ──┴── T4 ──┴── T3 ──┴── T2 ──┴── T1 ──┴────────┴────────┘
+    )
+};
```
Comment (dunk2k) [inline keyboards/pmk/recore/v3/keymaps/default/keymap.c:53]:
```suggestion
};

```

File: keyboards/pmk/recore/v3/keymaps/via/keymap.c [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+//├────────┼────────┼────────┼────────┼────────┼────────┴────────┴────────┤├────────┼────────┼────────┼────────┼────────┼────────┴────────┴────────┤
+   KC_TRNS ,KC_LEFT ,KC_LEFT ,KC_DOWN ,KC_RIGHT,KC_RIGHT,                                     KC_LEFT ,KC_LEFT ,KC_DOWN ,KC_RIGHT,KC_RIGHT,KC_NO   ,
+//├────────┼────────┼────────┼────────┼────────┼────────┬────────┬────────┤├────────┬────────┬────────┼────────┼────────┼────────┼────────┼────────┤
+   KC_TRNS ,KC_NO   ,KC_NO   ,KC_DOWN ,KC_NO   ,KC_NO   ,KC_TRNS ,KC_TRNS , KC_TRNS ,KC_TRNS ,KC_NO   ,KC_NO   ,KC_DOWN ,KC_NO   ,KC_NO   ,KC_NO   ,
+//├────────┼────────┼────────┼────────┼────────┼────────┼────────┼────────┘└────────┼────────┼────────┼────────┼────────┼────────┼────────┼────────┤
+   KC_TRNS ,KC_NO   ,KC_NO   ,KC_TRNS ,KC_TRNS ,KC_NO   ,KC_NO   ,                   KC_NO   ,KC_NO   ,KC_NO   ,KC_NO   ,KC_TRNS ,KC_NO   ,QK_CLEAR_EEPROM   
+//└────────┴────────┴── T1 ──┴── T2 ──┴── T3 ──┴── T4 ──┴── T5 ──┘                  └── T5 ──┴── T4 ──┴── T3 ──┴── T2 ──┴── T1 ──┴────────┴────────┘
+    )
+};
```
Comment (dunk2k) [inline keyboards/pmk/recore/v3/keymaps/via/keymap.c:53]:
```suggestion
};

```

File: keyboards/pmk/recore/v3/keymaps/via/rules.mk [hunk 6]
```diff
@@ -0,0 +1 @@
+VIA_ENABLE = yes
```
Comment (dunk2k) [inline keyboards/pmk/recore/v3/keymaps/via/rules.mk:1]:
```suggestion
VIA_ENABLE = yes

```

File: keyboards/pmk/recore/v3/rules.mk [hunk 7]
```diff
@@ -0,0 +1 @@
+SERIAL_DRIVER = vendor
```
Comment (dunk2k) [inline keyboards/pmk/recore/v3/rules.mk:1]:
```suggestion
SERIAL_DRIVER = vendor

```

**Review by dunk2k — CHANGES_REQUESTED** [review]

File: keyboards/pmk/recore/v3/keymaps/default/keymap.c (hunk 3 above)
Comment (dunk2k) [inline keyboards/pmk/recore/v3/keymaps/default/keymap.c:21]:
```suggestion
//├────────┼────────┼────────┼────────┼────────┼────────┴────────┴────────┘└────────┼────────┼────────┼────────┼────────┼────────┴────────┴────────┤
   KC_ESC  ,KC_A    ,KC_S    ,KC_D    ,KC_F    ,KC_G    ,                                     KC_H    ,KC_J    ,KC_K    ,KC_L    ,KC_SCLN ,KC_QUOT ,
//├────────┼────────┼────────┼────────┼────────┼────────┬────────┬────────┐┌────────┬────────┬────────┼────────┼────────┼────────┼────────┼────────┤
```

**Review by dunk2k — CHANGES_REQUESTED** [review]
Could this be applied to all other occurrences in both `default` and `via` keymaps

File: keyboards/pmk/recore/v3/keymaps/default/keymap.c [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+   KC_TRNS ,KC_NO   ,KC_NO   ,KC_UP   ,KC_NO   ,KC_NO   ,KC_NO   ,KC_NO   , KC_NO   ,KC_NO   ,KC_NO   ,KC_NO   ,KC_UP   ,KC_NO   ,KC_NO   ,KC_NO   ,
+//├────────┼────────┼────────┼────────┼────────┼────────┴────────┴────────┤├────────┼────────┼────────┼────────┼────────┼────────┴────────┴────────┤
+   KC_TRNS ,KC_LEFT ,KC_LEFT ,KC_DOWN ,KC_RIGHT,KC_RIGHT,                                     KC_LEFT ,KC_LEFT ,KC_DOWN ,KC_RIGHT,KC_RIGHT,KC_NO   ,
+//├────────┼────────┼────────┼────────┼────────┼────────┬────────┬────────┤├────────┬────────┬────────┼────────┼────────┼────────┼────────┼────────┤
+   KC_TRNS ,KC_NO   ,KC_NO   ,KC_DOWN ,KC_NO   ,KC_NO   ,KC_TRNS ,KC_TRNS , KC_TRNS ,KC_TRNS ,KC_NO   ,KC_NO   ,KC_DOWN ,KC_NO   ,KC_NO   ,KC_NO   ,
+//├────────┼────────┼────────┼────────┼────────┼────────┼────────┼────────┘└────────┼────────┼────────┼────────┼────────┼────────┼────────┼────────┤
+   KC_TRNS ,KC_NO   ,KC_NO   ,KC_TRNS ,KC_TRNS ,KC_NO   ,KC_NO   ,                   KC_NO   ,KC_NO   ,KC_NO   ,KC_NO   ,KC_TRNS ,KC_NO   ,QK_CLEAR_EEPROM
```
Comment (dunk2k) [inline keyboards/pmk/recore/v3/keymaps/default/keymap.c:50]: more human readable:
`KC_TRNS` -> `_______`
`KC_NO` -> `XXXXXXX`
`QK_CLEAR_EEPROM` -> `EE_CLR`
'KC_RIGHT' -> 'KC_RGHT`

```suggestion
   _______ ,KC_F1   ,KC_F2   ,KC_F3   ,KC_F4   ,KC_F5   ,_______ ,_______,  _______ ,_______ ,KC_F6   ,KC_F7   ,KC_F8   ,KC_F9   ,KC_F10  ,_______ ,
//├────────┼────────┼────────┼────────┼────────┼────────┼── X2 ──┼── X3 ──┤├── X3 ──┼── X2 ──┼────────┼────────┼────────┼────────┼────────┼────────┤
   _______ ,KC_EXLM ,KC_AT   ,KC_LCBR ,KC_RCBR ,KC_PIPE ,_______ ,_______,  _______ ,KC_RCBR ,KC_PSLS ,KC_P7   ,KC_P8   ,KC_P9   ,KC_PMNS ,_______ ,
//├────────┼────────┼────────┼────────┼────────┼────────┴────────┴────────┤├────────┼────────┼────────┼────────┼────────┼────────┴────────┴────────┤
   _______ ,KC_HASH ,KC_DLR  ,KC_LBRC ,KC_RBRC ,KC_GRV  ,                                      KC_PAST ,KC_P4   ,KC_P5   ,KC_P6   ,KC_PPLS ,_______ ,
//├────────┼────────┼────────┼────────┼────────┼────────┬────────┬────────┤├────────┬────────┬────────┼────────┼────────┼────────┼────────┼────────┤
   _______ ,KC_PERC ,KC_CIRC ,KC_LPRN ,KC_RPRN ,KC_TILD ,_______ ,_______ , _______ ,_______ ,KC_P0   ,KC_P1   ,KC_P2   ,KC_P3   ,KC_PENT ,_______ ,
//├────────┼────────┼────────┼────────┼────────┼────────┼────────┼────────┘└────────┼────────┼────────┼────────┼────────┼────────┼────────┼────────┤
   _______ ,_______ ,_______ ,_______ ,_______ ,_______ ,_______ ,                   KC_ENT  ,_______ ,_______ ,_______ ,KC_PDOT ,KC_PENT ,_______ 
//└────────┴────────┴── T1 ──┴── T2 ──┴── T3 ──┴── T4 ──┴── T5 ──┘                  └── T5 ──┴── T4 ──┴── T3 ──┴── T2 ──┴── T1 ──┴────────┴────────┘
    ),
    [_NAV] = LAYOUT(
//┌────────┬────────┬────────┬────────┬────────┬────────┬── L1 ──┬── X1 ──┐┌── X1 ──┬── L1 ──┬────────┬────────┬────────┬────────┬────────┬────────┐
   _______ ,XXXXXXX ,XXXXXXX ,KC_UP   ,XXXXXXX ,XXXXXXX ,XXXXXXX ,XXXXXXX , XXXXXXX ,XXXXXXX ,XXXXXXX ,XXXXXXX ,KC_UP   ,XXXXXXX ,XXXXXXX ,QK_BOOT ,
//├────────┼────────┼────────┼────────┼────────┼────────┼── X2 ──┼── X3 ──┤├── X3 ──┼── X2 ──┼────────┼────────┼────────┼────────┼────────┼────────┤
   _______ ,XXXXXXX ,XXXXXXX ,KC_UP   ,XXXXXXX ,XXXXXXX ,XXXXXXX ,XXXXXXX , XXXXXXX ,XXXXXXX ,XXXXXXX ,XXXXXXX ,KC_UP   ,XXXXXXX ,XXXXXXX ,XXXXXXX ,
//├────────┼────────┼────────┼────────┼────────┼────────┴────────┴────────┤├────────┼────────┼────────┼────────┼────────┼────────┴────────┴────────┤
   _______ ,KC_LEFT ,KC_LEFT ,KC_DOWN ,KC_RGHT ,KC_RGHT ,                                     KC_LEFT ,KC_LEFT ,KC_DOWN ,KC_RGHT ,KC_RGHT ,XXXXXXX ,
//├────────┼────────┼────────┼────────┼────────┼────────┬────────┬────────┤├────────┬────────┬────────┼────────┼────────┼────────┼────────┼────────┤
   _______ ,XXXXXXX ,XXXXXXX ,KC_DOWN ,XXXXXXX ,XXXXXXX ,_______ ,_______ , _______ ,_______ ,XXXXXXX ,XXXXXXX ,KC_DOWN ,XXXXXXX ,XXXXXXX ,XXXXXXX ,
//├────────┼────────┼────────┼────────┼────────┼────────┼────────┼────────┘└────────┼────────┼────────┼────────┼────────┼────────┼────────┼────────┤
   _______ ,XXXXXXX ,XXXXXXX ,_______ ,_______ ,XXXXXXX ,XXXXXXX ,                   XXXXXXX ,XXXXXXX ,XXXXXXX ,XXXXXXX ,_______ ,XXXXXXX ,EE_CLR
```

**Review by dunk2k — CHANGES_REQUESTED** [review]
Could this change also be applied to `via` keymap

File: keyboards/pmk/recore/v3/keymaps/default/keymap.c (hunk 8 above)
Comment (joliverMI) [reply keyboards/pmk/recore/v3/keymaps/default/keymap.c]: Thank you, this looks way better. I copied it into the via keymap as well.

File: keyboards/pmk/recore/v3/keymaps/default/keymap.c
Comment (dunk2k) [inline keyboards/pmk/recore/v3/keymaps/default/keymap.c:1]: For `default` and `via` keymaps:
- Could line 19 be copied to lines 32 and 45?
- Could line 21 be copied to lines 34 and 47?

**Review by dunk2k — CHANGES_REQUESTED** [review]

File: keyboards/pmk/recore/v3/keymaps/default/keymap.c [hunk 9]
```diff
@@ -0,0 +1,53 @@
...
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [_QWERTY] = LAYOUT(
+//┌────────┬────────┬────────┬────────┬────────┬────────┬── L1 ──┬── X1 ──┐┌── X1 ──┬── L1 ──┬────────┬────────┬────────┬────────┬────────┬────────┐
+   KC_GRV  ,KC_1    ,KC_2    ,KC_3    ,KC_4    ,KC_5   ,MO(_SYMB),MO(_NAV),MO(_NAV),MO(_SYMB),KC_6    ,KC_7    ,KC_8    ,KC_9    ,KC_0    ,KC_MINS ,
+//├────────┼────────┼────────┼────────┼────────┼────────┼── X2 ──┼── X3 ──┤├── X3 ──┼── X2 ──┼────────┼────────┼────────┼────────┼────────┼────────┤
+   KC_TAB  ,KC_Q    ,KC_W    ,KC_E    ,KC_R    ,KC_T    ,KC_LBRC ,KC_MPLY, KC_PAUS ,KC_RBRC , KC_Y    ,KC_U    ,KC_I    ,KC_O    ,KC_P    ,KC_EQL  ,
+//├────────┼────────┼────────┼────────┼────────┼────────┴────────┴────────┘└────────┼────────┼────────┼────────┼────────┼────────┴────────┴────────┤
+   KC_ESC  ,KC_A    ,KC_S    ,KC_D    ,KC_F    ,KC_G    ,                                     KC_H    ,KC_J    ,KC_K    ,KC_L    ,KC_SCLN ,KC_QUOT ,
+//├────────┼────────┼────────┼────────┼────────┼────────┬────────┬────────┐┌────────┬────────┬────────┼────────┼────────┼────────┼────────┼────────┤
```
Comment (dunk2k) [inline keyboards/pmk/recore/v3/keymaps/default/keymap.c:21]: can this change be applied to `via` keymap also?
```suggestion
//├────────┼────────┼────────┼────────┼────────┼────────┼────────┴────────┘└────────┴────────┼────────┼────────┼────────┼────────┴────────┴────────┤
   KC_ESC  ,KC_A    ,KC_S    ,KC_D    ,KC_F    ,KC_G    ,                                     KC_H    ,KC_J    ,KC_K    ,KC_L    ,KC_SCLN ,KC_QUOT ,
//├────────┼────────┼────────┼────────┼────────┼────────┼────────┬────────┐┌────────┬────────┼────────┼────────┼────────┼────────┼────────┼────────┤
```

**Review by dunk2k — CHANGES_REQUESTED** [review]

File: keyboards/pmk/recore/v3/keymaps/default/keymap.c [hunk 10]
```diff
@@ -0,0 +1,53 @@
...
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+
+enum my_layers {
+    _QWERTY,
+    _NAV,
+    _SYMB,
+};
+
```
Comment (dunk2k) [inline keyboards/pmk/recore/v3/keymaps/default/keymap.c:11]:
```suggestion
```

File: keyboards/pmk/recore/v3/keymaps/via/keymap.c [hunk 11]
```diff
@@ -0,0 +1,53 @@
...
+
+#include QMK_KEYBOARD_H
+
+enum my_layers {
+    _QWERTY,
+    _NAV,
+    _SYMB,
+};
+
+
```
Comment (dunk2k) [inline keyboards/pmk/recore/v3/keymaps/via/keymap.c:12]:
```suggestion
```

File: keyboards/pmk/recore/v3/readme.md [hunk 12]
```diff
@@ -0,0 +1,27 @@
+# recorev3
+
+![recorev3](https://i.imgur.com/28fVsP3.jpeg)
+
+The Posey Recore keyboard is an ortholinear keyboard designed from the gournd up for gaming. By pushing the left half of your split keyboard out of the way, you'll be able to game with better posture and increase mouse movement. The thumb cluster arc can be remapped to control and shift for use in gaming.
+
+* Keyboard Maintainer: [Javier Oliver](https://github.com/joliverMI)
+* Hardware Supported: recore_v3 and logic_module_v1
+* Hardware Availability: poseysmechanicalkeyboards.com
```
Comment (dunk2k) [inline keyboards/pmk/recore/v3/readme.md:9]:
```suggestion
* Hardware Availability: [poseysmechanicalkeyboards.com](https://poseysmechanicalkeyboards.com/)
```

**Review by dunk2k — CHANGES_REQUESTED** [review]

File: keyboards/pmk/recore/v3/info.json [hunk 13]
```diff
@@ -0,0 +1,143 @@
...
+    "bootloader": "rp2040",
+    "build": {
+        "debounce_type": "sym_defer_pk"
+    },
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "rgblight": true,
+        "command": false,
+        "console": false,
```
Comment (dunk2k) [inline keyboards/pmk/recore/v3/info.json:14]:
```suggestion
```

**Review by dunk2k — CHANGES_REQUESTED** [review]

**Review by dunk2k — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Pull request merged**